waspc/data/Generator/templates/server/src/routes/auth/passport/google/google.js

```diff
@@ -0,0 +1,77 @@
+import express from 'express'
+import passport from 'passport'
+import GoogleStrategy from 'passport-google-oauth20'
+
+import waspServerConfig from '../../../../config.js'
+import { contextWithUserEntity, authConfig, findOrCreateUserByExternalAuthAssociation } from '../../utils.js'
+import { sign } from '../../../../core/auth.js'
+import { configFn, getUserFieldsFn } from './googleConfig.js'
+
+// Validates the provided config function returns all required data.
+const config = ((config) => {
```

Comment:
What if this function throws an exception. Who can catch it?

Reply:
Good question. In this case, it will die. But we really don't have a way to recover here, since we need that config. So we could either:
- leave as is, let it blow up, and hope the stack trace points them to the problem in their code, or
- catch it, make an error message, and they would still need to find their problem and fix it

I slightly lead toward the first (current) only because it should help them find the issue more easily. But this is a good thing to think about in general - if we have user-defined code errors, and we cannot recover, do we just let it blow up or catch and provide a "better" error message in some way?